mmdet3d/models/fusion_layers/coord_transform.py

```diff
@@ -0,0 +1,144 @@
+import torch
+from functools import partial
+
+from mmdet3d.core.points import get_points_type
+
+
+class Coord3DTransformation():
+    r"""3D Coord transformation for 3D-2D fusion.
+    Use the operations in data augmentation pipeline.
+
+    Args:
+        dtype (torch.dtype): Dtype of matrix and vector.
+        device (torch.device): Device to save matrix and vector.
+        coords_type (str): 'DEPTH' or 'CAMERA' or 'LIDAR'
+        img_meta (dict): Meta info regarding data transformation.
+        pcd_rotate_mat (torch.Tensor): Point cloud rotation matrix.
+        pcd_scale_factor (float): Point cloud scale factor.
+        pcd_trans_factor (torch.Tensor): Point cloud translation vector.
+        pcd_horizontal_flip (bool):
+            Whether flip point cloud along horizontal direction.
+        pcd_vertical_flip (bool):
+            Whether flip point cloud along vertical direction.
+
+    Attributes:
+        coords_type (str): 'DEPTH' or 'CAMERA' or 'LIDAR'
+        pcd_rotate_mat (torch.Tensor): Point cloud rotation matrix.
+        pcd_scale_factor (float): Point cloud scale factor.
+        pcd_trans_factor (torch.Tensor): Point cloud translation vector.
+        pcd_horizontal_flip (bool):
+            Whether flip point cloud along horizontal direction.
+        pcd_vertical_flip (bool):
+            Whether flip point cloud along vertical direction.
+    """
+
+    def __init__(self,
+                 dtype,
+                 device,
+                 coords_type,
+                 img_meta=None,
+                 pcd_rotate_mat=None,
+                 pcd_scale_factor=None,
```

Comment:
Should use the default value for this.
`pcd_scale_factor=1.0, pcd_trans_factor=0, pcd_horizontal_flip=False, pcd_vertical_flip=False`.